### ISSUE TITLE
v1.44 with ub18-cuda11.4 - zero hashrate on AMD

### ISSUE THREAD

**Issue by tymoteuszrogalewski** [sp-hash/TeamBlackMiner#236]
Hi there.

Since this version I have zero hashrate on two testing machines in SimpleMining (AMD):

First rig:
1x Asus RX 6600 XT 8GB

Second rig:
6x RX 570 8GB

Third rig with NV 1070 works ok

upto v1.43 all works ok.

Second issue is that when stop v1.44 and run v1.43 - Ive got error that API port cannont be opened. Must reboot whole machine. Must some issue with v1.44

**Comment by sp-hash** [comment]
Can you show the logfile or screenshot of the minwer window please

**Comment by tymoteuszrogalewski** [comment]
The first issue: (rig with 6600 XT)
```
00:00:03 [2022-01-24 12:06:06.670] Job 97f310
00:00:03 [2022-01-24 12:06:08.254] Job de1b04
00:00:03 [2022-01-24 12:06:08.846] Job 723e70
00:00:03 [2022-01-24 12:06:08.966] Job 07a9e5
00:00:03 [2022-01-24 12:06:10.062] Job 477d75
00:00:03 [2022-01-24 12:06:10.839] Job c19803
00:00:03 [2022-01-24 12:06:10.927] Job ea3c58
00:00:03 [2022-01-24 12:06:10.983] Job 6a13c6
00:00:04 [2022-01-24 12:06:11.983] Job 64da16
00:00:04 [2022-01-24 12:06:12.007] Job 5f328d
00:00:04 [2022-01-24 12:06:12.070]
00:00:04 [2022-01-24 12:06:12.071]
00:00:04 [2022-01-24 12:06:12.071] eu1.ethermine.org (ethash)    PING: 0ms    VARDIFF: 1.00     EPOCH: 468
00:00:04 [2022-01-24 12:06:12.071]
00:00:04 [2022-01-24 12:06:12.071]
00:00:04 [2022-01-24 12:06:12.071] ID      BOARD    TYPE    KERN    XINT    TEMP    FAN     CORE    MEM     WATT
00:00:04 [2022-01-24 12:06:12.071] GPU0    gfx1032  OpenCL  0       71      0/0     0       0       0       0
00:00:04 [2022-01-24 12:06:12.071]                                                                          0
00:00:04 [2022-01-24 12:06:12.071]
00:00:04 [2022-01-24 12:06:12.071] ID      BOARD    HASHRATE/W    HASHRATE      AVERAGE       SHARES       RATE
00:00:04 [2022-01-24 12:06:12.071] GPU0    gfx1032  0.00 H/W      19.47 GH/s    19.26 GH/s    0/0/0        100.00%
00:00:04 [2022-01-24 12:06:12.071]                  0.00 H/W      19.47 GH/s    19.26 GH/s    0/0/0        100.00%
00:00:04 [2022-01-24 12:06:12.071]
00:00:04 [2022-01-24 12:06:12.071] SHARES PER MINUTE: 0.00                                POOL HASHRATE: 0.00 H/s
00:00:04 [2022-01-24 12:06:12.071]
00:00:04 [2022-01-24 12:06:12.919] Job 6a052f
00:00:04 [2022-01-24 12:06:14.048] Job 2a9957
00:00:04 [2022-01-24 12:06:14.079] Job 6fdf62
00:00:04 [2022-01-24 12:06:16.064] Job 7d4d2b
00:00:04 [2022-01-24 12:06:16.392] Job d09e58
00:00:04 [2022-01-24 12:06:16.424] Job 5cfebb
00:00:04 [2022-01-24 12:06:17.417] Job ce92f9
00:00:04 [2022-01-24 12:06:17.480] Job 6d8ac4
00:00:04 [2022-01-24 12:06:17.577] Job 6c90c0
00:00:04 [2022-01-24 12:06:18.489] Job eb5cae
00:00:04 [2022-01-24 12:06:18.705] Job 58af00
```

API tells:
```
miner@simplemining:~$ timeout 5 curl -s http://127.0.0.1:4068/summary
{"name":"TBMiner","version":"1.44","api_version":"1.3","num_gpu_threads":1,"total_hashrate":-2147483648,"total_accepted":0,"total_rejected":0,"total_stale":0,"uptime_minutes":5}{"url":"eu1.ethermine.org","port":4444,"worker":"","algo":"ethash","job":"303c66","epoch":468,"block":0,"diff":1,"ping":0}{"0":{"gpu":0,"pcie_id":3,"board_name":"gfx1032","type":"OpenCL","xintensity":71,"gpu_temp":0,"mem_temp":0,"fan":0,"core_clock":0,"mem_clock":0,"watt":0,"hashrate":-2147483648,"dag":true,"accepted":0,"rejected":0,"stale":0}}
```


The second issue:
`00:00:00 [2022-01-24 12:01:14.041] Cannot initialize API server. Port in use. Retrying in 6 secs
`
...but this error I see is also after v1.43 stop and v.144 start. Maybe there is something in killing old process - I will look at this closer a bit later.

**Comment by sp-hash** [comment]
Fixed in v1.45
